common/tor.c

```diff
@@ -58,175 +54,198 @@ struct tor_service_reaching {
 	size_t hlen;
 };
 
-
-static struct io_plan *io_tor_connect_create_onion_finished(struct io_conn *conn, struct tor_service_reaching *reach)
+static struct io_plan *io_tor_connect_create_onion_finished(struct io_conn
```

Comment:
Maybe better this way?

    static struct io_plan *
    io_tor_connect_create_onion_finished( args ...